config/locales/en.yml

```diff
@@ -24,7 +24,7 @@ en:
     flexible_working_info: 'Yes. Email %{mailto} for more information.'
     starts_on: 'Expected start date'
     ends_on: 'End date'
-    expired: 'This vacancy has expired'
+    expired: 'This job has expired'
```

Comment:
@gazaston  This is where I got stuck before - should this not be `This job vacancy has expired?` It does not read correctly as it is.

Reply:
Perhaps 'This job posting has expired' instead?

Reply:
Actually `job post` as `job vacancy` is even more confusing.